README.md

```diff
@@ -203,29 +223,41 @@ bash smoke_test.sh localhost
 ```
 
 ### Programs and Projects
```

Comment:
it might be useful to link to this section https://github.com/uc-cdis/fence/blob/master/docs/user.yaml_guide.md#programs-and-projects-crud-access

users must update the user.yaml to give themselves access to create programs/projects _before_ creating programs/projects

Reply:
done